cpp/ql/test/query-tests/Likely Bugs/Likely Typos/FutileParams/FutileParams.expected

```diff
@@ -0,0 +1,3 @@
+| test.c:7:3:7:5 | call to foo | This call has arguments, but $@ is not declared with any parameters. | test.c:1:6:1:8 | foo | foo |
+| test.c:13:3:13:19 | call to not_yet_declared1 | This call has arguments, but $@ is not declared with any parameters. | test.c:13:3:13:3 | not_yet_declared1 | not_yet_declared1 |
```

Comment:
We could (now or in a follow-up PR) modify `Function.getLocation()` so that an implicit declaration is not used as the location to show for the function itself if there is an alternative non-implicit declaration, as in this case?